src/transformers/commands/serving.py

```diff
@@ -361,7 +402,7 @@ def _serve(req: "ChatCompletionInput"):
 
             def stream_response(_inputs):
                 try:
-                    max_new_tokens = req.max_tokens or generation_config.max_new_tokens or 256
+                    max_new_tokens = req.max_tokens or generation_config.max_new_tokens or 1024
```

Comment:
Changed the default to a larger value: generations were too short for models that think (like qwen 3)